.github/workflows/tests.yml

```
@@ -8,11 +8,18 @@ on:
 jobs:
   Tests:
     name: ${{ matrix.os }} / ${{ matrix.python-version }}
-    runs-on: ${{ matrix.os }}-latest
+    runs-on: ${{ matrix.image }}
     strategy:
       matrix:
-        os: [Ubuntu, MacOS, Windows]
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        os: [Ubuntu, macOS, Windows]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11.0-rc.1"]
```

```suggestion
        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11.0-dev"]
```

`-dev` will point to the latest RC, unless you want to bump this manually every time.